client/components/login/login.test.js

```diff
@@ -392,4 +392,56 @@ describe("<Login /> interactions", () => {
     const setIsActiveMock = wrapper.instance().props.setIsActive.mock;
     expect(setIsActiveMock.calls.length).toBe(1);
   });
+  it("should show error toast when server error", () => {
```

Comment:
This test does not fail without the fix. Were you aware of it? What is the purpose of this test?

Reply:
I wrote it because when the internal server error is from the openwisp-radius then it shows an internal server error toast.
but when the API shows connection refused or timeout the error is generic i.e. login error occurred.
So I thought to add both tests.
Should I remove the internal server error one?

Reply:
It's ok